docs/architecture.md

```diff
@@ -20,26 +20,25 @@ A **Trace** is a data/execution path through the system, and can be thought of a
 This section details the constituents of Jaeger, and how they relate to each other. It is arranged by the order in which spans from your application interact with them. 
 
 ###Jaeger client
-Jaeger clients are language specific implementations of the OpenTracing API. The contain a Tracer, which is an interface for Span creation, and context propagation. 
+Jaeger clients are language specific implementations of the OpenTracing API. They contain a Tracer, which is an interface for Span creation, and context propagation. 
 Some common frameworks (e.g. dropwizard) are instrumented out of the box, making it easy to get started. 
 
-An instrumented library creates spans when receiving new requests, and attaches context information (trace id, and span id) to outgoing requests. Note that only ids are propagated with requests, and any other information like operation name, logs, etc is not propagated, and is simply collected at the service by the Jaeger client.
+An instrumented library creates spans when receiving new requests, and attaches context information (trace id, span id, and baggage) to outgoing requests. Note that only ids and baggage are propagated with requests, and any other information like operation name, logs, etc is not propagated, and is simply collected at the service by the Jaeger client.
```

Comment:
* s/and is simply collected at the service by the Jaeger client./it is transmitted out of process asynchronously, in the background, to Jaeger Agents, and only for sampled traces./